src/box/sql/vdbe.c

```diff
@@ -4370,6 +4370,24 @@ case OP_Init: {          /* jump */
 	goto jump_to_p2;
 }
 
+/* Opcode: GenFuncid P1 * * P4 *
```

Comment:
I think it should be `GenFuncId`, right? Also, I think it would be better to name this opcode `OP_NextFuncId`.